browser/profiles/brave_renderer_updater.cc

```diff
@@ -86,6 +88,11 @@ BraveRendererUpdater::BraveRendererUpdater(
       base::BindRepeating(&BraveRendererUpdater::UpdateAllRenderers,
                           base::Unretained(this)));
 #endif
+
+  pref_change_registrar_.Add(
+      playlist::kPlaylistEnabledPref,
```

Comment:
a possible improvement: could we just add items to a list of prefs and then call `pref_change_registrar_.Add(..)` in a for-loop avoiding copy-pasting?